README.md

```diff
@@ -1,2 +1,108 @@
-# To-Do-List-with-Webpack
-An interactive To-do list with Webpack
+<a name="readme-top"></a>
+
+# To-Do List Project
+
+# 📗 Table of Contents
+
+- [📖 About the To-Do List Project](#about-project)
+- [🛠 Built With](#built-with)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgments)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 To-do List Project <a name="about-project"></a>
+
+**This is the To-do List Project in order to show what we have learned so far at HTML, CSS and JS.**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **made with use of github workflow**
+- **made with use of linter checkers**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JS</li>
+  </ul>
+</details>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Coming Soon!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Portfolio presentation will be coming soon.
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project you need a Chrome, Safari or Mozilla browser
+
+### Setup <a name="setup"></a>
+
+Clone this repository to your desired folder: https://github.com/mertkantaroglu/To-Do-List-with-Webpack.git
+
+### Install <a name="install"></a>
+
+Install this project with Bash command
+
```

Comment:
- [ ] I would suggest please write the proper comment like `git clone ...........` and installation part like npm install and to run that project use npm start. please write the proper command so anyone can run you project on browser.